website/static/js/project-organizer.js

```diff
@@ -376,6 +377,13 @@ var tbOptions = {
     hScroll : 300,
     filterTemplate : function() {
         var tb = this;
+        function resetFilter () {
+            $osf.trackClick('myProjects', 'filter', 'clear-search');
+            tb.filterText('');
+            tb.resetFilter.call(tb);
+            tb.updateFolder(allTopLevelProjectsCache(), tb.treeData);
+            $('.db-poFilter>input').val('');
```

Comment:
Why do you need to change the value with jQuery? Isn't the input bound to a prop?